dvc/data_cloud.py

```diff
@@ -39,7 +39,9 @@ def get_remote(self, remote=None, command="<command>"):
         if remote:
             return self._init_remote(remote)
 
-        raise NoRemoteError(command)
+        has_any_remote = any(key.startswith("remote ") for key in self._config)
```

Comment:
This feels as a part of the logic inside exception itself. So everything should be either inside or outside.

Also, `key.startswith("remote ")` is a bit fishy, we check for regex in other places.

Reply:
Got it, changed that to use `list_options` as I've found elsewhere. Remote URLs are required so finding any means that there's a valid remote that could be used.